app/build.gradle

```diff
@@ -33,4 +33,5 @@ dependencies {
 
   //Third party libs
   implementation 'zachinio.choreographer:choreographer:1.0.5'
+  implementation 'jp.wasabeef:recyclerview-animators:3.0.0'
```

Comment:
The use of third parties libs for animations is not welcome for this challenge, blc it will not help you understand the underlining process of creating an animation.

Reply:
yeah !

Reply:
So remove the third party libs and implement the animation as shown via the course

Reply:
Good orientation @Ora-Kool